### DISCUSSION TITLE
How do we get OnlyM to work with a laptop, second monitor and TV screens?

### DISCUSSION THREAD

**Discussion by AllanChes** [AntonyCorbett/OnlyM#458] [Q&A] [answered]
We are using a small screen laptop and are trying to use a second monitor to give more screen space at the Sound Desk. So that the JW Library will display on the Hall Monitors, we have had to choose the second Sound Desk monitor as the "main screen" for the laptop, otherwise the JW Library displays on the second monitor, as does OnlyM. However, when we choose the second monitor as the "Main screen", the Library is fine and displays on the hall monitors but OnlyM's second screen disappears. When you mouse over OnlyM on the task bar, it is showing a second screen but it is just white and the second monitor loses it's image but nothing is displayed. If you click on the mouseover thumbnail on the taskbar, nothing happens - the image does not appear on any screen.
Is this a 'bug' or is there some other way to get it to operate with 3 screens?
Thanks for any advise.
Allan, Nuriootpa Congregation, South Australia
PS. I posted this question under 'Issues' sever weeks ago, but no reply yet so thought I would try here.

**Comment by AntonyCorbett** [comment, marked as answer]
Just a note that OnlyM allows you to select the monitor to use for media output (in the Settings page), but JWL does not. JWL's output monitor is always the one that appears to the immediate right of your primary monitor in Windows Display Settings (where you can also rearrange the logical position of multiple monitors).

**Reply by AllanChes** [reply]
Thanks. I had not noticed that option as it was greyed out as 'Windowed' was ticked. Works at home, so will be keen to check it out at the hall tomorrow.
Thanks again!
Allan

**Reply by AntonyCorbett** [reply]
Hi Allan. Yes, you should turn off "windowed" operation in OnlyM at the hall, and then select the appropriate monitor. Regards, Antony

**Reply by AllanChes** [reply]
Thanks again Anthony for your help!!
Can't wait to try it in the morning!
Regards,
Allan

**Reply by AllanChes** [reply]
Hi Anthony,
Worked perfectly. In OnlyM Settings I unticked 'Windowed' and chose the hall TV Screens as the OnlyM Monitor.  Now we have the Sound Desk's second monitor set to be the main screen, and it displays the JW Library and OnlyM, with Zoom full-screen on the laptop screen. Selecting images etc on JW Library or OnlyM now displays instantly on the hall monitors (as does Zoom when we drag its second screen up there) - Beautiful!
Thank you!

**Reply by AllanChes** [reply]
Hi again Anthony,
The JWLibrary and OnlyM are working fine on our 3 screen setup, but, because we had occasional hiccups with Zoom sharing the hall monitors, I installed JwlMediaWin. Now, when JWLibrary starts up, the yeartext appears normally on the hall monitors but then, a few seconds later, a small window with the yeartext appears briefly, on top of the library itself, then disappears. When it does, the text on the TV monitors increases in size and the JW logo under the text, gets pushed off the screens altogether, as does the bottom and right quarter of any image or video on JW Library or OnlyM. It almost appears as though it is trying to apply the lower resolution of the second monitor, to the hall monitors. I have had to disable it. Is our 3 screen set-up causing this or is there a setting somewhere I have missed?
Allan

**Reply by AntonyCorbett** [reply]
This is probably a limitation of using JwlMediaWin on a system that has monitors with different scaling settings, e.g. a main monitor with 125% scaling and a media display monitor with 100% scaling. It's something that JwlMediaWin should really work around, but currently, the only fix is to ensure that all monitors have the same scaling value (Windows System -> Display)

**Reply by AllanChes** [reply]
Thanks Anthony.
I will check the scaling settings in the morning.
Regards,
Allan
